e2e/node/resource.test.ts

```diff
@@ -146,28 +142,17 @@ describe("Authenticated end-to-end", () => {
     );
   });
 
-  it("can create, delete, and differentiate between RDF and non-RDF Resources", async () => {
-    const fileUrl = `${sessionResource}.txt`;
-
-    const sessionFile = await overwriteFile(
-      fileUrl,
-      Buffer.from("test"),
-      fetchOptions,
-    );
-    const sessionDataset = await getSolidDataset(sessionResource, fetchOptions);
-
-    expect(isRawData(sessionDataset)).toBe(false);
-    expect(isRawData(sessionFile)).toBe(true);
-
-    await deleteFile(fileUrl, fetchOptions);
-  });
-
-  it("can create, delete, and differentiate between RDF and non-RDF Resources using a node Buffer", async () => {
+  it("can create, delete, and differentiate between RDF and non-RDF Resources using a Blob from the node Buffer package", async () => {
     const fileUrl = `${sessionResource}.txt`;
 
     const sessionFile = await overwriteFile(
       fileUrl,
-      NodeBuffer.from("test"),
+      // We need to type cast because the buffer definition
+      // of Blob does not have the prototype property expected
+      // by the lib.dom.ts
```

Comment:
If I remember correctly, you had opened an issue about this a while ago. Do you happen to have the link to it?

Reply:
```suggestion
      // We need to type cast because the buffer definition
      // of Blob does not have the prototype property expected
      // by the lib.dom.ts
      // See https://github.com/microsoft/TypeScript/issues/53668
      // and https://github.com/microsoft/TypeScript/issues/52166
```

@NSeydoux could you leave a comment in https://github.com/microsoft/TypeScript/issues/53668 to see if we can get this re-opened?